packages/jmix-addon-data-tools/src/entity-inspector/EntityInspector.helpers.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  findEntityMetadata, 
+  MetaClassInfo, 
+  Metadata
+} from "@haulmont/jmix-react-core";
+import {EntityNamesInfo} from "./EntityInspector.types";
+
+export function getEntityNamesInfo(allEnttityNames: EntityNamesInfo[], entityName: string) : EntityNamesInfo {
+  return allEnttityNames.find((entityNamesItem) => {
```

Comment:
just an nitpick, but could be rewritten shortly 
```suggestion
  return allEnttityNames.find(entityNamesItem => entityNamesItem.entityName === entityName)!;
```